includes/editor-templates/panel.php

```diff
@@ -186,22 +186,22 @@
 		</div>
 	<# } #>
 
-	<?php if ( ! Utils::has_pro() ) : ?>
```

Comment:
This line is important. These promotions only show if pro is not active. These promotions are hidden once Pro licensing is active. We have separate promotions in the pro repo for essential to advanced promotions.

Reply:
@Barcov 

This condition checks if Elementor Pro in not installed. However, external addon developers need to check whether their Premium plugins are not installer.

Therefor, I removed this from the editor-template, and now it's part of `get_upsale_data()` method. You have set `condition` arguments, with different activation condition, to control when to display promotions.

Reply:
Check the condition bellow, we now check for `elementData.upsale_data.condition`.

Reply:
Love that!